index.html

```diff
@@ -42,6 +42,202 @@ <h4> I’am a Software Developer</h4>
             <li><a href="#"><img src="./images/Disabledm.svg" alt=""></a></li>
         </ul>
     </section>
+    <section class="jobs_container">
```

Comment:
- [ ] Kindly add `display:grid` to the main container wrapping all the projects as this is part of the project requirements 👍🏿 
- [ ] Kindly align all content of this div to the center this will make your design look more like the Figma design